sdk/tests/deqp/framework/opengl/gluShaderUtil.js

```diff
@@ -44,6 +44,11 @@ gluShaderUtil.GLSLVersion = {
 gluShaderUtil.getGLSLVersion = function(gl) {
     var glslversion = gl.getParameter(gl.SHADING_LANGUAGE_VERSION);
 
+    // TODO: Versions are not yet well implemented... Firefox returns GLSL ES 1.0 in some cases,
```

Comment:
Is this TODO addressed at this point in the various browsers? If not, let's please file bugs against them. Otherwise this will likely get lost.